app/scripts/lib/setupSentry.js

```diff
@@ -32,6 +32,12 @@ export const SENTRY_BACKGROUND_STATE = {
     accounts: false,
     currentBlockGasLimit: true,
   },
+  AccountsController: {
+    internalAccounts: {
+      accounts: true,
+      selectedAccount: true,
```

Comment:
Change to false

Reply:
Changed in 8e9da248ef551773ea52922f6359f2c41612a3da